domain/wollemi/service_format_test.go

```diff
@@ -226,6 +226,97 @@ func (t *ServiceSuite) TestService_GoFormat() {
 
 		require.NoError(t, wollemi.GoFormat(rewrite, []string{"app/server"}))
 	})
+
+	t.It("can resolve dependencies by import_path", func(t *T) {
+		data := t.GoFormatTestData()
+
+		delete(data.Stat, "third_party/go/github.com/spf13/BUILD.plz")
+		delete(data.Lstat, "third_party/go/github.com/spf13/BUILD.plz")
+		delete(data.Parse, "third_party/go/github.com/spf13/BUILD.plz")
```

Comment:
If the old build file was found and used instead of the new one that we created the test would still fail appropriately. So for the sake of simplicity I think we can just skip these deletes.